src/transport/TransportMgrBase.h

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
```

Comment:
nit: new file, likely should only be 2021 rather than the range 2020-2021 .

Reply:
The contents are moved from an old file, so the old year may be required for the situation.